NgForm incorrectly implements interface IFormController

The latest version of angular from DefinitelyTyped added a `$name` property to the `IFormController`. See DefinitelyTyped/DefinitelyTyped#10352

> ERROR in c:\Users\AC186067\Documents\Projects\ARCS\arcs-online\arcs-client\node_modules\ng-metadata\src\common\directives\ng_form.d.ts                                                              
> (1,31): error TS2420: Class 'NgForm' incorrectly implements interface 'IFormController'.                                                                                                            
>   Property '$name' is missing in type 'NgForm'.

Workaround is to use a prior version from DT. Such as the following:

```json
"globalDependencies": {
  "angular": "registry:dt/angular#1.5.0+20160627105203"
},
```

--- a/src/common/directives/ng_form.ts
+++ b/src/common/directives/ng_form.ts
@@ -8,6 +8,7 @@ export abstract class NgForm implements ng.IFormController {
   $invalid: boolean;
   $submitted: boolean;
   $error: any;
+  $name: string;
   $pending: any;
 
   $addControl( control: angular.INgModelController ): void {}

